### PR TITLE
update fluentd doc

### PR DESCRIPTION
**Pull request by lovebaby979**
update fluentd doc

### CODE REVIEW

File: how-to/how-to-import-kata-logs-with-fluentd.md [hunk 1]
```diff
@@ -185,7 +185,7 @@ in Kibana:
 ![Kata tags in EFK](./images/efk_syslog_entry_detail.png).
 
 We can however further sub-parse the Kata entries using the
-[Fluentd plugins](https://docs.fluentbit.io/manual/parser/logfmt) that will parse
+[Fluentd plugins](https://docs.fluentbit.io/manual/pipline/parser/logfmt) that will parse
```
Comment (liubin) [inline how-to/how-to-import-kata-logs-with-fluentd.md:188]: The correct URL should be `https://docs.fluentbit.io/manual/pipeline/parsers/logfmt`.

**Comment by liubin** [comment]
Hi @lovebaby979 , thank you for this PR. But for getting this PR to be merged, you may need do some more:

- [create an issue](https://github.com/kata-containers/documentation/issues/new/choose) to describe what you will fix.
- re-write a more meaningful [commit message](https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#patch-format)
- make sure the `continuous-integration/travis-ci/pr` check has passed.

File: how-to/how-to-import-kata-logs-with-fluentd.md (hunk 1 above)
Comment (lovebaby979) [reply how-to/how-to-import-kata-logs-with-fluentd.md:188]: OK @liubin

**Comment by fidencio** [comment]
This was superseded by https://github.com/kata-containers/documentation/pull/789

**Pull request closed without merging**